05-JWT-Basics/starter/errors/custom-error.js

```diff
@@ -1,7 +1,6 @@
 class CustomAPIError extends Error {
-  constructor(message, statusCode) {
+  constructor(message) {
```

Comment:
Another way you might see this done is the status code is set to INTERNAL SERVER ERROR here and then overridden in the classes that inherit like `bad-request`. 

Neither's necessarily better, just giving another option